docs/en/Running-Alluxio-On-Docker.md

```diff
@@ -141,12 +141,32 @@ $ cd opt/alluxio
 $ bin/alluxio runTests
 ```
 
-### Sharing ramdisk with clients
+### Read/Write to the local worker 
 
+There are two options to read/write to the local worker efficiently. The option that shares ramdisk
+with the client requires the memory limit of the client container to be higher than the amount of
+data it writes to the local ramdisk. Whereas the domain socket approach does not have this
+limitation but it consumes more CPU.
+
+#### Share ramdisk with clients
 Running the worker with `-v /mnt/ramdisk:/mnt/ramdisk` shares the ramdisk between the worker
 and host. To make this ramdisk available to clients in other containers running on the same host,
 those containers should also be run with `-v /mnt/ramdisk:/mnt/ramdisk`.
 
+#### Enable domain socket
```

Comment:
maybe make this "Share domain socket with clients" to be parallel to the ramdisk section